### DISCUSSION TITLE
I don't understand the position data. Please help.

### DISCUSSION THREAD

**Discussion by DanieldeQuervain** [br-g/openf1#8] [Q&A] [answered]
If i look at the position data for example the driver max van stappen. At the first timestamp he is in position 1 and then loses a position in every subsequent timestamp of the race. After position 20 it jumps to position 3 and counts down again.
here is the link:
 https://api.openf1.org/v1/position?meeting_key=1142&driver_number=1 
 
 Is "position" not the drivers position during the race defined by the meeting_key? 
 Thanks for the help. Cheers

**Comment by br-g** [comment]
Hello,

`meeting_key` is for the full GP week-end, including practice and quali.
If you are looking for position during the race, you should use `session_key`.

For example, here is the position of Verstappen during the Saudi GP race: [https://api.openf1.org/v1/position?session_key=7779&driver_number=1](https://api.openf1.org/v1/position?session_key=7779&driver_number=1)

Cheers

**Comment by DanieldeQuervain** [comment, marked as answer]
Great! That does answer my question. Thanks for the help! 👍
